### PR TITLE
BLOCKS-268 Add missing bricks in Catblocks and update language

### PR DESCRIPTION
**Pull request by liflyanna**
*Please enter a short description of your pull request and add a reference to the Jira ticket.*
https://jira.catrob.at/browse/BLOCKS-268
Add new missing bricks from Catroid to Catblocks. Update languages from Catroid as well.
### Your checklist for this pull request
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Make sure you use BLOCKS instead of CATROID in your commit message
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed (Actions)
- [x] Post a message in the *#catblocks* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/library/js/blocks/categories/look.js [hunk 1]
```diff
@@ -621,5 +621,86 @@ export default {
...
+        name: 'brick_additive_particle_effect_spinner_INFO'
+      }
+    ]
+  },
+  SetCameraFocusPointBrick: {
+    message0: '%{BKY_SET_CAMERA_FOCUS}',
+    args0: [
+      {
+        type: 'field_input',
+        name: 'HORIZONTAL_FLEXIBILITY',
```
Comment (bprattes) [inline src/library/js/blocks/categories/look.js:668]: Please change name to brick_set_camera_focus_horizontal_input

File: src/library/js/blocks/categories/look.js [hunk 2]
```diff
@@ -621,5 +621,86 @@ export default {
...
+        text: 'unset'
+      },
```
Comment (bprattes) [inline src/library/js/blocks/categories/look.js:670]: Please add _INFO field for each input.

File: src/library/js/blocks/categories/look.js [hunk 3]
```diff
@@ -621,5 +621,86 @@ export default {
...
+      {
+        type: 'field_input',
+        name: 'VERTICAL_FLEXIBILITY',
```
Comment (bprattes) [inline src/library/js/blocks/categories/look.js:673]: Please change name to brick_set_camera_focus_vertical_input

File: src/library/js/blocks/categories/look.js [hunk 4]
```diff
@@ -621,5 +621,86 @@ export default {
...
+        text: 'unset'
+      },
+      {
+        type: 'field_image',
+        src: `${document.location.pathname}media/info_icon.svg`,
+        height: 24,
+        width: 24,
+        alt: '(i)',
+        flip_rtl: true,
+        name: 'brick_set_camera_focus_INFO'
```
Comment (bprattes) [inline src/library/js/blocks/categories/look.js:683]: Please change name to brick_set_camera_focus_vertical_input_INFO

File: i18n/strings_to_json_mapping.json [hunk 5]
```diff
@@ -82,6 +83,8 @@
   "LOOKS_DELETE_LOOK": "${brick_delete_look}",
   "LOOKS_PAINT_NEW_LOOK": "${brick_paint_new_look} %1%2",
   "LOOKS_EDIT_LOOK": "${brick_edit_look}",
+  "SET_CAMERA_FOCUS": "${brick_focus_point} %1 ${brick_horizontal_and} %2 ${brick_vertical_flex} %3",
```
Comment (bprattes) [inline i18n/strings_to_json_mapping.json:86]: Please fix arguments (%1%2 for brick_set_camera_focus_horizontal_input and %3%4 for brick_set_camera_focus_vertical_input) and add percent symbol (compare with [Catroid](https://github.com/Catrobat/Catroid/blob/1d2962b31c1483e1edc9b429460e77c0858cf4ae/catroid/src/main/res/layout/brick_set_camera_focus.xml))

**Review by bprattes — CHANGES_REQUESTED** [review]

File: src/library/js/blocks/categories/look.js [hunk 6]
```diff
@@ -621,5 +621,86 @@ export default {
...
+      }
+    ]
+  },
+  SetParticleColorBrick: {
+    message0: '%{BKY_SET_PARTICLE_COLOR}',
+    args0: [
+      {
+        type: 'field_input',
+        name: 'COLOR',
```
Comment (bprattes) [inline src/library/js/blocks/categories/look.js:692]: Please change name to brick_set_color_edit_text

**Review by bprattes — CHANGES_REQUESTED** [review]

File: i18n/strings_to_json_mapping.json (hunk 5 above)
Comment (liflyanna) [reply i18n/strings_to_json_mapping.json:86]: unfortunately this does not work as they are defined as following in the language.xml files
 <string name="brick_focus_point">Become focus point with</string>
    <string name="brick_horizontal_and">horizontal and</string>
    <string name="brick_vertical_flex">vertical flexibility</string>

**Review by bprattes — APPROVED** [review]
LGTM

**Pull request merged**